packages/override-tools/src/Cli.ts

```diff
@@ -330,31 +328,31 @@ function printValidationErrors(errors: Array<ValidationError>) {
 
   if (filesMissing.length > 0) {
     const errorMessage =
-      "Found override files that aren't listed in the manifest. Overrides can be added to the manifest by using 'yarn override add <override>':";
+      "Found override files that aren't listed in the manifest. Overrides can be added to the manifest by using 'yarn override add <override>' (where override is package relative):";
     console.error(chalk.red(errorMessage));
     filesMissing.forEach(err => console.error(` - ${err.file}`));
     console.error();
   }
 
   if (overridesMissing.length > 0) {
     const errorMessage =
-      "Found overrides in the manifest that don't exist on disk. Remove existing overrides using 'yarn override remove <override>':";
+      "Found overrides in the manifest that don't exist on disk. Remove existing overrides using 'yarn override remove <override>' (where override is package relative):";
     console.error(chalk.red(errorMessage));
     overridesMissing.forEach(err => console.error(` - ${err.file}`));
     console.error();
   }
 
   if (baseFilesNotFound.length > 0) {
     const errorMessage =
-      "Found overrides whose original files do not exist. Remove existing overrides using 'yarn override remove <override>':";
+      "Found overrides whose original files do not exist. Remove existing overrides using 'yarn override remove <override>' (where override is package relative):";
```

Comment:
>relative [](start = 151, length = 8)

So, this is kind of annoying. `yarn` will lie about cwd and always pretend you're running the script from the package directory. `npx`  doesn't run into this, but will also go download the `override` npm package if one isn't found, and we've seen issues before where `yarn` doesn't link the `override` bin correctly without a force or clean install,.